hack/generator/pkg/astmodel/identifier_factory.go

```diff
@@ -45,11 +56,29 @@ func NewIdentifierFactory() IdentifierFactory {
 	return &identifierFactory{
 		renames:       createRenames(),
 		reservedWords: createReservedWords(),
+		idCache:       make(idCache),
```

Comment:
I didn't know you could pass the variable instead of the type - nice!

Reply:
that is a type :)

Reply:
Suddenly I don't like it so much. 😁 🤔